WORKSPACE

```diff
@@ -316,15 +316,34 @@ npm_install(
     package_json = "//examples/hellohttp/nodejs:package.json",
 )
 
-# error_prone_annotations required by protobuf 3.7.1
-maven_jar(
-    name = "error_prone_annotations_maven",
-    artifact = "com.google.errorprone:error_prone_annotations:2.3.2",
+RULES_JVM_EXTERNAL_TAG = "2.9"
+
+RULES_JVM_EXTERNAL_SHA = "e5b97a31a3e8feed91636f42e19b11c49487b85e5de2f387c999ea14d77c7f45"
```

Comment:
Could you please put these values into the target, without defining the variables.
I believe Renovate (which we use for this repo) won't be able to detect updates for the target below when it consumes these variables.

Reply:
done.